network/svi.go

```diff
@@ -122,12 +122,9 @@ func (c evpnClientImpl) UpdateSvi(ctx context.Context, name string, updateMask [
 	}
 	defer closer()
 	client := c.getEvpnSVIClient(conn)
-	svi, err := client.GetSvi(ctx, &pb.GetSviRequest{
+
```

Comment:
can we extract that change in `svi.go` file into a separate commit like `fix(network/svi): create svi directly for update`